scheduler/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package scheduler orders tasks in distro queues.
```

Comment:
Is it worth mentioning the different scheduler implementations?

Reply:
I don't think so, since we only support a single one.